stdlib/public/core/StringObject.swift

```diff
@@ -989,7 +1019,21 @@ extension _StringObject {
       countAndFlags: storage._countAndFlags)
 #endif
   }
-
+  
+#if arch(arm64)
+  internal init(
+    taggedConstantCocoa cocoa: AnyObject, length: Int
```

Comment:
Either put isASCII in the parameter list or update the label to denote that assumption.